packages/manager/modules/pci/src/projects/project/storages/containers/containers.html

```diff
@@ -82,6 +91,19 @@
                     ></span>
                 </oui-action-menu-item>
             </oui-action-menu>
+            <oui-datagrid-column
+                data-title="'pci_projects_project_storages_containers_public_label' | translate"
+                data-property="public"
+                data-sortable
+                data-filterable
+            >
+                <oui-switch
+                    model="$value"
+                    disabled="$ctrl.publicToggleLoading"
```

Comment:
Are we sure that we will have to disable this toggle for all the rows, when one of the rows is being toggled?